community/xpra/APKBUILD

```diff
@@ -27,7 +28,7 @@ prepare() {
 }
```

Comment:
You can remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.